packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -187,6 +179,13 @@ export default function Sidebar({
     },
   ];
 
+  const myFeedMenuItem: SidebarMenuItem = {
+    icon: <ListIcon Icon={HomeIcon} />,
+    title: 'My feed',
+    path: '/my-feed',
```

Comment:
Should we go with `my-feed` or `myFeed`?

Reply:
I was thinking from an URL point of view `my-feed` would look nicer, so we could use that?
Let me know what you and @sshanzel think.

Reply:
Have the same opinion as Chris for the URL pov.